### PR TITLE
Add hotfix instructions to DWDS

### PR DESCRIPTION
**Pull request by elliette**
Follow-up to https://github.com/dart-lang/webdev/pull/1867

### CODE REVIEW

File: dwds/CONTRIBUTING.md [hunk 1]
```diff
@@ -138,3 +138,42 @@ you need to:
...
+master branch. Instead you only want to apply a fix to the current version of
+DWDS in Flutter.
+
+### Instructions:
+
+1. In the Github UI's
+   [commit history view](https://github.com/dart-lang/webdev/commits/master),
+   find the commit that prepared the release of DWDS that you would like to
+   hotfix.
+1. Click on `< >` ("Browse the repository at this point in history").
```
Comment (annagrin) [inline dwds/CONTRIBUTING.md:156]: 2? :)

File: dwds/CONTRIBUTING.md [hunk 2]
```diff
@@ -138,3 +138,42 @@ you need to:
...
+   from `commit_hash`".
+1. From your local clone of DWDS, run `git fetch upstream`. (*Note: this assumes
+   you have already configured git to sync your fork with the `upstream`
+   repository. If you haven't, follow
+   [these instructions](https://docs.github.com/en/get-started/quickstart/fork-a-repo#configuring-git-to-sync-your-fork-with-the-upstream-repository).)*
+1. Search for the branch that you just created, e.g.
+   `git branch -a | grep 16.0.2-hotfix-release`
+1. Track that branch with `git checkout --track branch_name` (e.g.
+   `remotes/upstream/16.0.2-hotfix-release`)
+1. You can now make the change you would like to hotfix on the branch you have.
```
Comment (annagrin) [inline dwds/CONTRIBUTING.md:169]: We should add instructions on how to modify the CI for the hotfix branch before adding the actual fix (move testing on irrelevant sdk versions, add testing on relevant sdk versions if needed, remove sending notifications, etc)

**Review by annagrin — COMMENTED** [review]
Thanks Elliott, I left some comments

File: dwds/CONTRIBUTING.md (hunk 1 above)
Comment (annagrin) [inline dwds/CONTRIBUTING.md:156]: Can we add general description of the steps, so it is clear what are we trying to do? The steps above could move under the general step as substeps,  for example

1. create a branch off the release that needs a hotfix
   a. In the Github UI's
   [commit history view](https://github.com/dart-lang/webdev/commits/master),
   find the commit that prepared the release of DWDS that you would like to
   hotfix. 
  b. Click on `< >` ("Browse the repository at this point in history").
  ...
2. Update CI for the new branch 
  a ...
3. Add the fix

Comment (elliette) [reply dwds/CONTRIBUTING.md]: This is a side affect of running the autoformatter - it still gets rendered as a numeric sequence (1, 2, 3, etc)

File: dwds/CONTRIBUTING.md (hunk 2 above)
Comment (elliette) [reply dwds/CONTRIBUTING.md]: Done!

File: dwds/CONTRIBUTING.md (hunk 1 above)
Comment (elliette) [reply dwds/CONTRIBUTING.md]: Done!

File: dwds/CONTRIBUTING.md [hunk 3]
```diff
@@ -138,3 +138,58 @@ you need to:
...
+   assumes you have already configured git to sync your fork with the `upstream`
+   repository. If you haven't, follow
+   [these instructions](https://docs.github.com/en/get-started/quickstart/fork-a-repo#configuring-git-to-sync-your-fork-with-the-upstream-repository).)_
+
+   e. Search for the branch that you just created, e.g.
+   `git branch -a | grep 16.0.2-hotfix-release` f. Track that branch with
+   `git checkout --track branch_name` (e.g.
+   `remotes/upstream/16.0.2-hotfix-release`)
+
+1. Update the CI tests so that the branch tests against the appropriate Dart
```
Comment (annagrin) [inline dwds/CONTRIBUTING.md:176]: 2. ?

**Review by annagrin — APPROVED** [review]
LGTM with small change request - the sequential numbering is still showing all 1's, can we update it to 1, 2, 3...?

**Comment by elliette** [comment]
> LGTM with small change request - the sequential numbering is still showing all 1's, can we update it to 1, 2, 3...?

This is a change made by the Markdown auto-formatter. You can create a sequentially ordered list using all 1s. That way MD automatically handles the numbering.

**Pull request merged**